src/components/views/rooms/RoomInfoLine.tsx

```diff
@@ -45,8 +45,7 @@ const RoomInfoLine: FC<IProps> = ({ room }) => {
     const membership = useMyRoomMembership(room);
     const memberCount = useRoomMemberCount(room);
 
-    const elementCallVideoRoomsEnabled = useFeatureEnabled("feature_element_call_video_rooms");
-    const isVideoRoom = room.isElementVideoRoom() || (elementCallVideoRoomsEnabled && room.isCallRoom());
+    const isVideoRoom = useIsVideoRoom(room);
```

Comment:
Should this skip the feature flag check too? It's used in the room preview card which prompts the user to opt into the beta.

Reply:
Good spot